### DISCUSSION TITLE
ScriptReference/PrefabUtility.RevertPropertyOverride

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#523] [Notes]
# ScriptReference/PrefabUtility.RevertPropertyOverride

https://docs.unity3d.com/ScriptReference/PrefabUtility.RevertPropertyOverride.html

**Comment by joonjoonjoon** [comment]
observed this running very (unusably) slow on a large batch of operations (~10.000)
https://forum.unity.com/threads/prefabutility-revertpropertyoverride-takes-1s-per-operation.1393330/
